src/renderer/components/dialogs/ViewGroup.tsx

```diff
@@ -217,9 +217,15 @@ function ViewGroupInner(props: {
     null
   )
 
+  if (viewMode === 'profile' && !profileContact) {
```

Comment:
you could just check for profile contact and remove the view mode, then you don't need this if statement and have simpler code?

Reply:
the thing is that if view mode is profile, profile contact must be non-null. If it's null, it's a bug. The code shows the main view anyway, but this error log is printed to console.

Reply:
my idea here was to remove the view mode and make it dependent on wether `profileContact` is set or not, so one variable less